src/main/java/at/hannibal2/skyhanni/api/HotmAPI.kt

```diff
@@ -0,0 +1,113 @@
+package at.hannibal2.skyhanni.api
+
+import at.hannibal2.skyhanni.data.HotmData
+import at.hannibal2.skyhanni.data.ProfileStorageData
+import at.hannibal2.skyhanni.utils.InventoryUtils
+import at.hannibal2.skyhanni.utils.ItemCategory
+import at.hannibal2.skyhanni.utils.ItemUtils.getItemCategoryOrNull
+import at.hannibal2.skyhanni.utils.NEUInternalName.Companion.asInternalName
+import at.hannibal2.skyhanni.utils.SkyBlockItemModifierUtils.getDrillUpgrades
+import at.hannibal2.skyhanni.utils.StringUtils.firstLetterUppercase
+import at.hannibal2.skyhanni.utils.TimeLimitedCache
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import net.minecraft.item.ItemStack
+import kotlin.time.Duration.Companion.seconds
+
+object HotmAPI {
+
+    fun copyCurrentTree() = HotmData.storage?.deepCopy()
+
+    val activeMiningAbility get() = HotmData.abilities.firstOrNull { it.enabled }
+
+    private val blueGoblinEgg = "goblin_omelette_blue_cheese".asInternalName()
```

Comment:
```suggestion
    private val blueGoblinEgg = "GOBLIN_OMELETTE_BLUE_CHEESE".asInternalName()
```

I know before it would get converted to this anyways but it should probably just be like this always